scripts/storybook/webpack.config.js

```diff
@@ -92,13 +92,5 @@ module.exports = (/** @type {webpack.Configuration} */ config) => {
 
   config.optimization.minimize = false;
 
-  if (process.env.TF_BUILD) {
```

Comment:
This log was to debug an issue that's no longer happening (something about northstar types getting included in random storybook builds sometimes)